python/3d/auto_generated/qgs3dmapsettings.sip.in

```diff
@@ -348,6 +348,20 @@ Sets whether to show camera's view center as a sphere (for debugging)
 Returns whether to show camera's view center as a sphere (for debugging)
 
 .. versionadded:: 3.4
+%End
+
+    void setShowCameraRotationCenter( bool enabled );
+%Docstring
+Sets whether to show camera's rotation center as a sphere (for debugging)
+
+.. versionadded:: 3.22
```

Comment:
This should be 3.24 if not being backported, (but I'm in favour of backporting these improvements)